### DISCUSSION TITLE
Stöð 2

### DISCUSSION THREAD

**Discussion by zupersponge** [iptv-org/community#685] [Q&A]
**Channel Name:** Stöð 2
**Channel ID:** Stod2.is
**Country:** Iceland
**Language:** Icelandic
**Website:** stod2.is
**Stream URL (optional):**
**Notes (optional):**

**Comment by BellezaEmporium** [comment]
DRM protected streams.

**Comment by zupersponge** [comment]
Ah, right

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19680
